QueryBuilder/Compilers/Compiler.cs

```diff
@@ -82,12 +83,13 @@ private Query TransformAggregateQuery(Query query)
             var outerClause = new AggregateClause()
             {
                 Columns = new List<string> { "*" },
-                Type = clause.Type
+                Type = clause.Type,
+                Alias = clause.Alias,
             };
 
             return new Query()
                 .AddComponent("aggregate", outerClause)
-                .From(query, $"{clause.Type}Query");
+                .From(query, $"{clause.Alias}{clause.Type.First().ToString().ToUpper()}{clause.Type.Substring(1)}Query");
```

Comment:
ToUpperInvariant, also could you add a comment as to what the line does?

Reply:
pushed an update